Bottom: 141529ed2a37cc6bcca798d16726424fbad9f6fc
Top:    5d4282e90682cc1e32ba1a2c63af01ebc9279b60
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-03-27 16:27:29 +0100

Refresh of add-support-for-depth-to-trian

---

diff --git a/src/hid/gtk/gui-output-events.c b/src/hid/gtk/gui-output-events.c
index df0446f..eb63d93 100644
--- a/src/hid/gtk/gui-output-events.c
+++ b/src/hid/gtk/gui-output-events.c
@@ -222,10 +222,6 @@ ghid_note_event_location (GdkEventButton * ev)
 {
   gint event_x, event_y;
   gboolean moved;
-  /* FIXME: IFDEF HACK */
-#ifdef ENABLE_GL
-  extern float global_depth; /* FIXME: REMOVE THIS HACK! */
-#endif
 
   if (!ev)
     {
@@ -241,7 +237,7 @@ ghid_note_event_location (GdkEventButton * ev)
   /* FIXME: IFDEF HACK */
 #ifdef ENABLE_GL
   /* Unproject event_x and event_y to world coordinates of the plane we are on */
-  ghid_unproject_to_z_plane (event_x, event_y, global_depth,
+  ghid_unproject_to_z_plane (event_x, event_y, 0,
                              &event_x, &event_y);
 #endif
